server/src/test/java/org/opensearch/cluster/metadata/IndexMetadataTests.java

```diff
@@ -88,6 +92,36 @@ protected NamedXContentRegistry xContentRegistry() {
         return new NamedXContentRegistry(IndicesModule.getNamedXContents());
     }
 
+    // Create the index metadata for a given index, with the specified version.
+    private static IndexMetadata createIndexMetadata(final Index index, final long version) {
```

Comment:
nitpick on code refactoring:

```
private static IndexMetadata createIndexMetadata(final Index index, final long version) {
    return createIndexMetadata(index, version, false (or the default value));
}
```

unless setting to default vs not setting the `.system()` achieves a different functionality?

Reply:
Pushed this change to reuse the method with default.